src/pages/ContentManager.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useMemo, useState } from "react";
+
+import SaveIcon from "@mui/icons-material/Save";
+import SearchIcon from "@mui/icons-material/Search";
+import { Box, Container, MenuItem } from "@mui/material";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import { makeStyles } from "@mui/styles";
+
+import ActionMenu from "~/components/ActionMenu";
+import DynamicModal from "~/components/Modal";
+import Tab from "~/components/tabs/Tab";
+import Tabs from "~/components/tabs/Tabs";
+import { useGlobalSetting } from "~/contexts/GlobalSettingProvider";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    marginBottom: theme.spacing(2)
+  },
+  searchContainer: {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    gap: theme.spacing(10)
+  }
+}));
+
+const MAPPING = { 0: "page_config", 1: "content_manager", 2: "document" };
+
+function ContentManager() {
+  const [activeTab, setActiveTab] = React.useState(1);
+  const { setting } = useGlobalSetting();
+  const { labels } = setting;
+  const [modelOpen, setModelOpen] = useState(false);
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleTabChange = useCallback(
+    (event, newValue) => {
+      setActiveTab(newValue);
+    },
+    [setActiveTab]
+  );
+
+  const renderTabContent = useMemo(() => {
+    return function render() {
```

Comment:
Please note it's not a correct implementation of useMemo. Please follow the correct implementation